scripts/comparison-test.sh

```diff
@@ -6,14 +6,7 @@ set -eo pipefail
 SCRIPT_DIR="$(cd "$(dirname "$0")" && pwd)"
 cd "$SCRIPT_DIR" || exit 1
 
-DATA="zq-sample-data"
-DATA_REPO="https://github.com/brimsec/zq-sample-data.git"
-if [ -d "$DATA" ]; then
-  (cd "$DATA"&& git pull) || exit 1
-else
-  git clone --depth=1 "$DATA_REPO"
-fi
```

Comment:
It would be fine with me to replace line 9 as you did and retain lines 10-15. I agree line 16 can be deleted.

Reply:
If you don't care either way, I'll leave it as-is with the excess lines removed and hence encourage running it via `make`.